account_export_csv/wizard/account_export_csv.py

```diff
@@ -0,0 +1,354 @@
+# Copyright 2013 Camptocamp SA
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import itertools
+import tempfile
+from io import StringIO, BytesIO
+import base64
+
+import csv
+import codecs
+
+from odoo import api, fields, models, _
+
+
+class AccountingWriter(object):
+
+    """
+    A CSV writer which will write rows to CSV file "f",
+    which is encoded in the given encoding.
+    """
+
+    def __init__(self, f, dialect=csv.excel, encoding="utf-8", **kwds):
+        # Redirect output to a queue
+        self.queue = StringIO()
+        # created a writer with Excel formating settings
+        self.writer = csv.writer(self.queue, dialect=dialect, **kwds)
+        self.stream = f
+        self.encoder = codecs.getincrementalencoder(encoding)()
+
+    def writerow(self, row):
+        # we ensure that we do not try to encode none or bool
+        row = (x or '' for x in row)
+        self.writer.writerow(row)
+        # Fetch UTF-8 output from the queue ...
+        data = self.queue.getvalue()
+        # ... and reencode it into the target encoding as BytesIO
+        data = self.encoder.encode(data)
+        # write to the target stream
+        self.stream.write(data)
+        # empty queue with seek() instead of truncate()
+        # see https://stackoverflow.com/a/9729516
+        # also problems with seek() if next line is shorter than previous
+        # chars of previous line are kept in the new one
+        self.queue.flush()
```

Comment:
We might want to just reinitialize the StringIO

According to https://stackoverflow.com/questions/4330812/how-do-i-clear-a-stringio-object
we could even simply create a new StringIO

```suggestion
        self.queue = StringIO()
```

Reply:
Done